cms/djangoapps/contentstore/proctoring.py

```diff
@@ -39,7 +39,10 @@ def register_special_exams(course_key):
         # if feature is not enabled then do a quick exit
         return
 
-    course = modulestore().get_course(course_key)
+    try:
+        course = modulestore().get_course(course_key)
```

Comment:
@Qubad786 It's really a good question. But I don't know where would be the course, not exists.
I have checked the spunk and I have found the error here, that's why I have added this defensive code here.